src/api/status/public/js/github-stats.js

```diff
@@ -0,0 +1,68 @@
+const fetchGitHubApi = (owner, repo, path) => {
```

Comment:
If your arrow function immediately returns the result of an expression, you don't need a body or `return`:

```js
const fetchGitHubApi = (owner, repo, path) =>
  fetch(`https://api.github.com/repos/${owner}/${repo}/${path}`, {
    headers: { Accept: 'application/vnd.github.v3+json' },
  });
```

Reply:
fixed!